### ISSUE TITLE
[Block Widgets] Long text gives a '414 request-uri too large' error when loading classic editor

### ISSUE THREAD

**Issue by Greatdane** [Automattic/wp-calypso#55653]
### Steps to Reproduce

1. Load a simple site with classic widgets already active, or an Atomic site with the classic widget plugin
2. Add a text widget and enter lots of text (bug came with 205 lines of text)
3. Save the text widget
4. Disable classic widget plugin, and go back to My Site → Appearance → Customize → Widgets
5. The widget will now show '414 request-uri too large'

### What I expected to happen

At step 5 above, I expect to see the text as usual, not an error message. 

Note, clicking on the error message will load the text without issue, but this is not clear at first sight.

### What actually happened

When opening the Customizer and choosing the widget, the error '414 request-uri too large' appears.

See here: 
<img width="889" alt="Markup 2021-08-23 at 16 51 09" src="https://user-images.githubusercontent.com/5075840/130410673-b7517c8f-b1a9-457f-a4a6-146bf59a0874.png">

### Context

Customer report: 4238136-zd-woothemes

I can reproduce this on any theme with a sidebar widget area, not tested in other widget areas. 

Note, this only seems to happen when user already had content in a classic widget. Brand new sites that have only used block widgets are not affected. 
I guess it has something to do with the **classic editor**. 

### Operating System

macOS

### Browser

Chrome/Chromium

### Is this specific to applied theme? If so, what is the theme name?

Tested on Edin and Penscratch 2

### Simple/Atomic

Simple, Atomic

### Number of Users Impacted

All users

### Available Workarounds

Installing classic widget plugin
Clicking on the error will also bring up the text.

### Reproducibility

Consistent

**Comment by Robertght** [comment]
I can't replicate this on a self-hosted website, but I noticed there's a similar error message here(although the context is different):

https://github.com/WordPress/gutenberg/issues/12226

**Comment by rickmgithub** [comment]
Same problem on 4302060-zd

Video of issue here - https://d.pr/v/9EV7Cs

Switched back to the classic widgets for the time being.

**Comment by ankurparihar** [comment]
Yeah, facing the same, how do we switch to classic widgets @rickmgithub , I'm new to wordpress.
